quinn-proto/src/transport_parameters.rs

```diff
@@ -418,6 +434,10 @@ impl TransportParameters {
             || params.max_udp_payload_size.0 < 1200
             || params.initial_max_streams_bidi.0 > MAX_STREAM_COUNT
             || params.initial_max_streams_uni.0 > MAX_STREAM_COUNT
+            || params.min_ack_delay.map_or(false, |min_ack_delay| {
+                // min_ack_delay uses nanoseconds, whereas max_ack_delay uses milliseconds
+                min_ack_delay.0 > params.max_ack_delay.0 * 1_000_000
```

Comment:
Isn't it microseconds, not nanoseconds?

Reply:
Good catch! Not sure how I got that one wrong... The draft is pretty clear about it:

> Note that while the endpoint's max_ack_delay transport parameter is in milliseconds ([Section 18.2](https://rfc-editor.org/rfc/rfc9000#section-18.2) of [[QUIC-TRANSPORT](https://www.rfc-editor.org/rfc/rfc9000)]), min_ack_delay is specified in microseconds